### ISSUE TITLE
BpmnError doesn't handle for ReactorTaskListener

### ISSUE THREAD

**Issue by EugeneMsv** [camunda/camunda-bpm-reactor#90]
I'm using your plugin and when the ReactorTaskListener throws BpmnError into notify method it doesn't handled. I can't catch it using task_error element into bpmn model, it just rollbacks transaction.

**Comment by jangalinski** [comment]
Yes, indeed, you are right. I actually never considered this behavior, but it is plausible since the CamundaEventbus is configured using the UncaughtErrorHandler which throws runtime exceptions ... 

A possible fix would be to filter on bpmn errors.

I will mark this as a bug.
